### PR TITLE
Translation Update for v1.9

### PR DESCRIPTION
**Pull request by MaxiderZocker**
Hello,
I've updated the Translation in accordance with the v1.9 Language file.
Would love to see this Implemented.
Furthermore, could you please Update the Language file to the v1.10 version, so that I can continue with the Translation.

Greetings,
Max

### CODE REVIEW

**Comment by CODEYXone** [comment]
Hey @MaxiderZocker 👋 Thanks a bunch for opening your first pull request! That's awesome. I'll take a look at it soon and provide feedback. Keep up the great work!

> Furthermore, could you please Update the Language file to the v1.10 version, so that I can continue with the Translation.

I'll get right on that once I've merged your PR to avoid any potential merge conflicts. Been there, not fun! 😄

**Comment by MaxiderZocker** [comment]
Alright, Thank you

File: game.po [hunk 1]
```diff
@@ -34,82 +34,82 @@ msgstr "="
...
 msgctxt "editor_sel_only_buildings"
 msgid "Keep only buildings in the selection"
-msgstr "Mantener solo los edificios en la selección"
+msgstr "Nur Gebäude in der Auswahl behalten"
 
 #v1.9.17
 msgctxt "editor_sel_only_child_tracks"
 msgid "Keep only parallel child tracks in the selection"
-msgstr "Mantener solo las vías paralelas hijas en la selección"
+msgstr "Nur parallele Kindergleise in der Auswahl behalten"
```
Comment (TheGamerH18) [inline game.po:92]: Maybe we should use something like "untergeordnete Gleise" instead of "Kindergleise"

File: game.po [hunk 2]
```diff
@@ -124,92 +124,92 @@ msgstr "NOT IN GAME - Interpolate tangent: average previous track and current tr
...
 msgctxt "editor_tip_cont_cancel_click"
 msgid "Click to cancel"
-msgstr "Click para cancelar"
+msgstr "Zum Abbrechen klicken"
 
 #v1.9.17
 msgctxt "editor_tip_cont_cancel_dbl_click"
 msgid "Double click to cancel"
-msgstr "Doble click para cancelar"
+msgstr "DOppelklick zum Abbrechen"
```
Comment (TheGamerH18) [inline game.po:132]: Should be "Doppelklick [...]"

**Comment by CODEYXone** [comment]
@MaxiderZocker I would like to implement some small changes and fixes. Could you add me as a contributor to your fork?

**Comment by MaxiderZocker** [comment]
Added you

File: game.po (hunk 1 above)
Comment (MaxiderZocker) [reply game.po:92]: yeah. I couldn't think of anything better than Kinder at that point. 
That is why you always should have different ppl looking over what you have done, to verify if it makes sense or not what you have written.

**Comment by CODEYXone** [comment]
Hey @MaxiderZocker, quick update: I'm halfway through the read. Planning to wrap up over the weekend. By the way, I noticed a few typos and some inconsistent capitalization. Would be great if you could fly over the text real quick and fix any inconsistencies you spot.

File: game.po (hunk 2 above)
Comment (CODEYXone) [reply game.po]: Resolved with 0cea2856a245fe98db0b586f1e5d4f299ac7771d

File: game.po (hunk 1 above)
Comment (CODEYXone) [reply game.po]: Resolved with 0cea2856a245fe98db0b586f1e5d4f299ac7771d

**Comment by MaxiderZocker** [comment]
Hey. Small question. Is this still being looked into, or not?
Greetings Max.

**Comment by CODEYXone** [comment]
LGTM

**Pull request merged**